connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -338,6 +339,38 @@ protected Connector getConnector(String connType) {
         }
     }
 
+    /**
+     * Checks a given {@link ConfigInfos} for validation error messages and adds an exception
+     * to the given {@link Callback} if any were found.
+     * @param config ConfigInfos to Read Errors from
+     * @param callback Callback to add Config Error Exception to
+     * @return TRUE iff Errors were Found in the Config
+     */
+    protected final boolean maybeAddConfigErrors(final ConfigInfos config,
```

Comment:
@kkonstantine The argument for `final` on method arguments is that you shouldn't reassign these variables (which is different from whether you mutate the values or not).